packages/fxa-settings/src/components/DeviceInfoBlock/index.tsx

```diff
@@ -57,7 +57,7 @@ export const DeviceInfoBlock = ({
   };
 
   return (
-    <div className="mt-9">
+    <div className="mt-9 mb-4">
```

Comment:
oh good catch!

Reply:
Something about this margin-top looks a little tall (I guess it doesn't quite center the device name in vertical space?) but I don't think we need to address it because I think it's like this in prod!

Reply:
Right? I matched parity but it did feel quite tall.